source/examples/rapids-coiled-cudf/notebook.ipynb

```diff
@@ -0,0 +1,2461 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    
</pre><p>Do we need this if the data is public on coiled s3? for example this example on their docs does not include the creds part https://docs.coiled.io/user_guide/uber-lyft.html nut I might be wrong. I think you can just pass to the read parquet <code>storage_options={"anon": True}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>